include/quicr/quicr_server.h

```diff
@@ -6,9 +6,9 @@
 #include <thread>
 #include <vector>
 
-#include <quicr/message_buffer.h>
-#include <quicr/quicr_common.h>
-#include <transport/transport.h>
+#include "message_buffer.h"
+#include "quicr_common.h"
+#include "transport/transport.h"
```

Comment:
Maybe we should move these includes ito the .cpp file and forward declare any types we get from the files? Might not be able to do that with transport but certainly with MessageBuffer

Reply:
I wonder what caused <quicr/quicr_common.h> to move to "quicr_common.h" .. IIRC the CMakeLists ensures that anything within include/quicr be included as <quicr/<some-header>

Reply:
That must have been some auto add.  I"ll correct that.